### PR TITLE
s-nail 14.9.13

### PR DESCRIPTION
**Pull request by alexanderharm**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
==> make build
  GEN uman.1
  GEN urc.rc
  LINK s-nail
Undefined symbols for architecture x86_64:
  "_main", referenced from:
     implicit entry/start for main executable
ld: symbol(s) not found for architecture x86_64
```

**Comment by alexanderharm** [comment]
Can anyone help me with this issue? Both the upstream developer and I myself are completely at loss about where the problem lies. Funnily enough I also fail building the previous version `14.9.11` now. When I run `clang -v` I receive the following output:

```
 "/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ld" -demangle -lto_library /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/libLTO.dylib -dynamic -arch x86_64 -dynamic -macosx_version_min 10.13.0 -o s-nail -L/usr/local/opt/openssl/lib -liconv -lssl -lcrypto -search_paths_first -lkrb5 -lidn2 -lcurses -lSystem /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/clang/10.0.0/lib/darwin/libclang_rt.osx.a
Undefined symbols for architecture x86_64:
  "_main", referenced from:
     implicit entry/start for main executable
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[1]: *** [s-nail] Error 1
make: *** [build] Error 2
```

**Comment by alexanderharm** [comment]
Fixed the problem. It was due to the standard Apple awk leaking file descriptors. Pulled in brew's awk as dependency and now everything works again.

File: Formula/s-nail.rb [hunk 1]
```diff
@@ -11,14 +11,12 @@ class SNail < Formula
     sha256 "4d137ea6f6ff75f6bd792584aa4d86ead5445f0e5df9c9ed7074d84df3b7fb0f" => :el_capitan
   end
 
+  depends_on "awk"
```
Comment (fxcoudert) [inline Formula/s-nail.rb]: Can we make it a build-only dependency?

```suggestion
  depends_on "awk" => :build
```

Comment (alexanderharm) [reply Formula/s-nail.rb]: Changed as requested.

**Comment by fxcoudert** [comment]
Thanks @alexanderharm

**Pull request closed without merging**